src/main/java/org/openmicroscopy/ms/zarr/RequestHandlerForImage.java

```diff
@@ -763,7 +775,7 @@ private void returnImageMasksDirectory(HttpServerResponse response, List<String>
         final ImmutableList.Builder<String> contents = ImmutableList.builder();
         contents.add(".zattrs");
         if (labeledMasks != null) {
-            contents.add("labeled/");
+            contents.add(DEFAULT_LABEL_NAME);
```

Comment:
Still needs a terminal "/"?

Reply:
Part of me says we should support _both_ but the removal was unintentional. Thanks.